pyproj/transformer.py

```diff
@@ -180,14 +242,45 @@ class Transformer:
 
     """
 
-    def __init__(self, base_transformer: Optional[_Transformer] = None) -> None:
-        if not isinstance(base_transformer, _Transformer):
+    def __init__(
+        self,
+        transform_init: Union[
+            TransformerFromCRS, TransformerFromPipeline, _Transformer, None
+        ] = None,
+    ) -> None:
+        if not isinstance(
+            transform_init, (_Transformer, TransformerFromCRS, TransformerFromPipeline)
+        ):
             ProjError.clear()
             raise ProjError(
                 "Transformer must be initialized using: "
                 "'from_crs', 'from_pipeline', or 'from_proj'."
             )
-        self._transformer = base_transformer
+
+        self._local = TransformerLocal()
+        if isinstance(transform_init, _Transformer):
+            # this is from TransformerGroup
+            # and should only be used from there
+            self._transform_init: Union[
+                TransformerFromCRS, TransformerFromPipeline
+            ] = TransformerFromPipeline(transform_init.definition)
```

Comment:
After thinking on it, I am not sure I like this solution. I am thinking making an `UnsafeTransformer` class for the `TransformerGroup` where it always returns the original transformer + a warning in the docstrings about not being thread-safe would be a better approach.